src/main/java/ToDos.java

```diff
@@ -0,0 +1,10 @@
+public class ToDos extends Task {
+    public ToDos(String toDosName){
```

Comment:
the name of the "toDosName" might be a bit confusing. Do you think there is a better name for it?